owlkettle/widgets.nim

```diff
@@ -28,15 +28,24 @@ when defined(nimPreviewSlimSystem):
 import gtk, widgetdef, cairo, widgetutils
 
 when defined(owlkettleDocs) and isMainModule:
-  echo "# Widgets\n\n"
+  echo """
+# Widgets
+
+## Custom Widgets
+When writing your application or custom widgets you will typically be writing a `viewable` (see [internals.md](https://github.com/can-lehmann/owlkettle/blob/main/docs/internals.md) for more information). All `viewables` have the following available to them by default:
+  - BaseWidget-fields and setters
```

Comment:
This is not correct. Viewables just inherit from `Widget`, not BaseWidget. (Even though the name `BaseWidget` would suggest that)

Reply:
In that case I think it makes sense to scratch that idea for a "Custom Widgets" section in the widgets-docs from me I think.
Maybe that makes more sense as part of the internals.md file as part of explaining what features a Viewable has etc.

Reply:
At this point the internals.md section is getting a lot of extra content pumped into it that provides this kind of information and this section has been removed from widgets.nim. I'll thusly mark this as resolved.